bundle/manifests/kepler-operator.clusterserviceversion.yaml

```diff
@@ -25,7 +25,7 @@ metadata:
     capabilities: Basic Install
     categories: Monitoring
     containerImage: quay.io/sustainable_computing_io/kepler-operator:0.6.2
-    createdAt: "2023-08-28T09:36:02Z"
+    createdAt: "2023-08-30T03:55:14Z"
```

Comment:
should this date change with every PR? or change only from release Job in CI?

Reply:
it is okay to change this whenever we run bundle update.